src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -390,29 +386,38 @@ const StreamActions = createReactClass({
                 </a>
               </Tooltip>
             </div>
-          </div>
-          <div className="hidden-sm stream-actions-assignee col-md-1" />
-          <div className="stream-actions-level col-md-1 hidden-xs" />
-          <div className="hidden-sm hidden-xs stream-actions-graph col-md-2">
-            <ToolbarHeader className="stream-actions-graph-label">
-              {t('Graph:')}
-            </ToolbarHeader>
-            <ul className="toggle-graph">
-              <li className={this.props.statsPeriod === '24h' ? 'active' : ''}>
-                <a onClick={this.selectStatsPeriod.bind(this, '24h')}>{t('24h')}</a>
-              </li>
-              <li className={this.props.statsPeriod === '14d' ? 'active' : ''}>
-                <a onClick={this.selectStatsPeriod.bind(this, '14d')}>{t('14d')}</a>
-              </li>
-            </ul>
-          </div>
-          <ToolbarHeader className="stream-actions-count align-right col-md-1 col-sm-2 col-xs-2">
-            {t('Events')}
-          </ToolbarHeader>
-          <ToolbarHeader className="stream-actions-users align-right col-md-1 col-sm-2 col-xs-2">
-            {t('Users')}
-          </ToolbarHeader>
-        </Toolbar>
+          </ActionSet>
+          <Box w={130} mx={2} className="hidden-xs">
```

Comment:
Do these need to be the same dimensions as each stream item? If so, can we create common components for each "column" so that their dimensions are handled together?